common/src/jvmMain/kotlin/Platform.kt

```diff
@@ -0,0 +1,7 @@
+package dev.kord.common
+
+public actual object Platform {
+    public actual val IS_JVM: Boolean = true
+    public actual val IS_NODE: Boolean = false
+    public actual val IS_BROWSER: Boolean = false
```

Comment:
```suggestion
    public actual const val IS_JVM: Boolean = true
    public actual const val IS_NODE: Boolean = false
    public actual const val IS_BROWSER: Boolean = false
```
will then be inlined